src/makeCalculator.js

```diff
@@ -38,6 +38,42 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function(num) {
+      this.result -= num;
+
+      return this;
+    },
+    divide: function(num) {
```

Comment:
```suggestion
    },
    
    subtract: function(num) {
      this.result -= num;

      return this;
    },
    
    divide: function(num) {
```
It's recommended to add empty lines between methods to make your code easier to read